UI/media-slider.cpp

```diff
@@ -51,3 +63,34 @@ bool MediaSlider::eventFilter(QObject *obj, QEvent *event)
 
 	return QSlider::eventFilter(obj, event);
 }
+
+int MediaSlider::posToRangeValue(QMouseEvent *event)
+{
+	QStyleOptionSlider opt;
+	initStyleOption(&opt);
+
+	const QRect groove = style()->subControlRect(
+		QStyle::CC_Slider, &opt, QStyle::SC_SliderGroove, this);
+	const QRect handle = style()->subControlRect(
+		QStyle::CC_Slider, &opt, QStyle::SC_SliderHandle, this);
+
+	int pos, sliderMin, sliderMax, handleLength;
```

Comment:
```suggestion
	int pos;
	int sliderMin;
	int sliderMax;
	int handleLength;
```
Also, move this to the beginning of the function with the other declarations.

Reply:
Fixed